faqs/optifine.md

```diff
@@ -0,0 +1,27 @@
+---
+title: OptiFine Compatibility
+search: optifine
+---
+OptiFine is known to be [incompatible with many mods, including CC: Tweaked](<https://github.com/sp614x/optifine/issues/7395>). If you are experiencing problems with peripherals not connecting, and you have OptiFine installed, you will need to remove OptiFine.
+
+There are many replacement mods available for OptiFine's functionality. You don't need all of these—you can pick just the ones you need.
+
+### Forge/NeoForge
+- **[Embeddium](<https://modrinth.com/mod/embeddium/versions>): performance improvements**
+- [Oculus](<https://modrinth.com/mod/oculus>): shaders
+- [Fusion](<https://modrinth.com/mod/fusion-connected-textures>): connected textures
+- [ETF](<https://modrinth.com/mod/entitytexturefeatures>): advanced resource pack texture features
+- [EMF](<https://modrinth.com/mod/entity-model-features>): custom entity models
+- [RyoamicLights](<https://modrinth.com/mod/ryoamiclights>): dynamic lights
+
+### Fabric/Quilt
+- **[Sodium](<https://modrinth.com/mod/sodium>): performance improvements**
+- **[Indium](<https://modrinth.com/mod/indium>): mod compatibility, required with Sodium**
+- [Sodium Extra](<https://modrinth.com/mod/sodium-extra>): more settings for Sodium
+- [Iris](<https://modrinth.com/mod/iris>): shaders
+- [Continuity](<https://modrinth.com/mod/continuity>): connected textures
+- [Animatica](<https://modrinth.com/mod/animatica>): advanced resource pack texture features
+- [CEM](<https://modrinth.com/mod/cem>): custom entity models
+- [LambDynamicLights](<https://modrinth.com/mod/lambdynamiclights>): dynamic lights

```

Comment:
I think we should probably *just* link the shader mods here, and leave everything else to the OF alternatives page. I'm always wary of having FAQs be too long. Sodium/Iris should work on NeoForge nowadays, so probably worth preferring those over Embeddium.

Maybe something like:

> For performance improvements, we recommend Sodium (for Fabric and NeoForge) or Embeddium (Forge). For shaders, Iris (Fabric/NeoForge) or Occulus (Forge) are recommended.
> 
> For Optifine's other features, view the full list of alternatives here.

Probably can be made a little less clunky.